getting_started_with_acceptance_tests.md

```diff
@@ -0,0 +1,80 @@
+
+> You can read more about them in this blog post: [Refactoring Legacy Code](https://github.com/ubiqua/engblog/blob/master/refactoring_legacy_code.md).
+
+One of the tools we chose for our plan of action is [Cucumber](https://cucumber.io/), and in this blogpost I will guide you through the basics and good practices of writing acceptance tests with it.
+
+## Acceptance Tests
+An acceptance test demonstrates what a feature (or system) needs to do in order for the stakeholders to find it acceptable. At Ubiqua, we like to think of testing in the following way:
+
+- Acceptance tests: build the right thing.
+- Unit tests: build the thing right.
+
+Cucumber allows us to write acceptance tests by specifying a concrete number of executable examples (build the right thing) of how the system should behave under certain circumstances. Writing down these examples encourages a team to think about the different use-cases that a feature needs to serve which in turn assists in revealing insights about other feature use-cases that were unknown at the beginning of the iteration. Such practices help to avoid a [solution looking for a problem](http://xyproblem.info/), and consecutively boost the team's productivity.
+
+## Overview of a Cucumber Feature
+A Cucumber feature is a set of examples of how the system should behave under certain circumstances. Each feature is made of a number of examples called ``Scenarios``. Scenarios must be independent and usually follow this pattern:
+
+- Context: Put the system in a particular state (``Given``).
+- Action: Do something with the system (``When``).
+- Expectation: Examine the new state is correct (``Then``).
+
+Features in Cucumber are written in ``Gherkin``. Gherkin is a language used to facilitate communication with stakeholders to illustrate what the software should do. The greatest advantage of using Gherkin is that it is designed for human readability and thus helps developers to interact with stakeholders and create documentation both at the same time.
+
+## A Cucumber Feature Example
+
+A simple Cucumber feature usually consists of the following components:
+
+- ``Feature``: A convenient place to put a summary and documentation about the group of tests that follow.
+- ``Scenario``: A concrete example of the desired behavior of the system under certain circumstances.
+- A list of ``Steps``: Steps are plain language instructions of what the system/user will do and the desired output it should create. It allows developers to pass parameters of the values that are relevant to the scenario by using regular expressions.
+
+Let's go ahead an write a simple Cucumber feature to verify that a user is able to send an order with some products to a specific client:
+
+``` gherkin
+Feature: Create an Order
+
+  Ubiqua allows users to order products for their clients.
+
+  Scenario: Order Some Products
+    Given I'm logged in as user "Diego" assigned to "Farmacia Moreno", with the following products:
```

Comment:
I think `as user "Diego"` is an incidental detail for this scenario. Maybe use `as a salesman`